src/components/DisbursementDetails/index.tsx

```diff
@@ -81,10 +85,17 @@ export const DisbursementDetails: React.FC<DisbursementDetailsProps> = ({
     isFetching: isWalletAssetsFetching,
   } = useAssetsByWallet(details.wallet.id);
 
+  const {
+    data: verificationsTypes,
+    error: verificationsTypesError,
+    isFetching: isVerificationTypesFetching,
```

Comment:
Nit: naming is a bit inconsistent. We should either use `verificationTypes...` (singular `verification`) or `verificationsTypes` (plural `verifications`).